x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/update_rules_route.ts

```diff
@@ -53,6 +54,8 @@ export const updateRulesRoute = (router: SecuritySolutionPluginRouter, ml: Setup
         });
         throwAuthzError(await mlAuthz.validateRuleType(request.body.type));
 
+        checkDefaultRuleExceptionListReferences({ exceptionLists: request.body.exceptions_list });
```

Comment:
Also here, same question about catching the error and returning with `siemResponse.error()`